File: C++/Codes/Assignment/Factorial.cpp
#include <iostream>
using namespace std;

main()
{
    int no;

    cout << "Enter the No: ";

    cin >> no;

    int fact = 1;

    for (int i = 1; i <= no; i++)
    {

        fact *= i;
    }
    cout << "The factorial is : " << fact;

    return 0;
}

File: Java/DSA/Recursion/Factorial.java
package Java.DSA.Recursion;

import java.util.Scanner;

public class Factorial {
    
    public static int fact(int a){
        if(a == 0) return 1;
        if(a == 1) return 1;

        return a * fact(a - 1);
    
    }
    public static void main(String[] args) {
        int a;
        Scanner sc = new Scanner(System.in);

        a = sc.nextInt();

        System.out.println(fact(a));
    }

}
